Fix clogged pump jack high idle power consumption

bluefire-fx opened the pull request:
When the pump jack can't output its fluid to an already filled fluid tank it stops working as intended. However it still consumes ~1024 flux/tick while doing nothing. This prevents the pump jack from consuming power by using the boolean "active" that gets only set true once adjacent fluid containers accepted some amount of fluid greater than zero.

bluefire-fx committed:
Fix clogged pump jack high idle power consumption

When the pump jack can't output its fluid to an already filled fluid tank it stops working as intended. However it still consumes ~1024 flux/tick while doing nothing. This prevents the pump jack from consuming power by using the boolean "active" that gets only set true once adjacent fluid containers accepted some amount of fluid greater than zero.

TwistedGate commented on src/main/java/flaxbeard/immersivepetroleum/common/blocks/tileentities/PumpjackTileEntity.java:
```diff
@@ -210,7 +209,10 @@ public void tick(){
 					}
-					
+					if(active)
+					{
```
Apart from the missing empty line above the `if(active)` and the bracket being on the same line its good

bluefire-fx replied:
Thats greate to hear, changed the file to match the requested coding style.

TwistedGate reviewed (commented):
How that flew under my radar is beyond me lol

bluefire-fx committed:
Fix clogged pump jack high idle power consumption

When the pump jack can't output its fluid to an already filled fluid tank it stops working as intended. However it still consumes ~1024 flux/tick while doing nothing. This prevents the pump jack from consuming power by using the boolean "active" that gets only set true once adjacent fluid containers accepted some amount of fluid greater than zero.
(edit to match coding style of repository)

TwistedGate commented:
Perfection.

Thank you! :)
